express-spotify-app/app.js

```diff
@@ -0,0 +1,73 @@
+const SpotifyWebApi = require('spotify-web-api-node');
+// Remember to paste here your credentials
+const clientId = 'a36d34bce1954db1b5b32e164c1ccf43',
+    clientSecret = '0315373a3a444091a4314b78759f447c';
+
+const spotifyApi = new SpotifyWebApi({
+  clientId : clientId,
+  clientSecret : clientSecret
+});
+
+const express = require('express');
+const app = express();
+const expressLayouts = require('express-ejs-layouts');
+
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+app.set('layout', 'layouts/main-layout');
+
+
+app.use(express.static('public'));
+app.use(expressLayouts);
+
+// Retrieve an access token.
+spotifyApi.clientCredentialsGrant()
```

Comment:
This piece of code is asynchronous, so you can have problems if the response is not ready when the app needs it. You can try to wrap it in a function with a callback.